src/main/java/com/backbase/ct/bbfuel/healthcheck/HealthCheck.java

```diff
@@ -13,6 +13,8 @@
 public class HealthCheck {
 
     private GlobalProperties globalProperties = GlobalProperties.getInstance();
+    private static final String ACTUATOR = "/actuator";
+    private static final String INFO = "/info";
```

Comment:
Can we make this configurable? It will be different for different environments (k8s, pcf, aws)

Reply:
Which paths should I hit for k8s and pcf?

Reply:
That's depending on the configuration, by default those endpoints are not exposed or direct ip calls.

Reply:
But then how I'm going to make it configurable per service? Then we will have 9 more boxes on the jenkins job, will look a little bit ugly.

Reply:
We can make it configurable globally or as the service id. The k8s triggers will have a new trigger job for them. We can add them in the additional argument fields. If we want we can merge this, or make it configurable globally. I can add a pr later for k8s support.

Reply:
I think that would be better. Afaik, pipelines that use these healthcheck endpoints are the AWS test pipelines.

Reply:
Agreed, let's make it configurable for now. Will the default be compatible for 2.17.0 or 2.16.4?

Reply:
Default will be compatible with 2.17.0, I will make the configuration